src/sentry/api/endpoints/internal_register_trusted_relay.py

```diff
@@ -0,0 +1,98 @@
+from datetime import datetime, timezone
+
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log, features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.authentication import UserAuthTokenAuthentication
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers.models.organization import TrustedRelaySerializer
+from sentry.models.options.organization_option import OrganizationOption
+from sentry.utils.audit import create_audit_entry
+
+
+@region_silo_endpoint
+class InternalRegisterTrustedRelayEndpoint(OrganizationEndpoint):
```

Comment:
The default permissions from `OrganizationEndpoint` will allow `manager` role users to make changes, is that ok for this endpoint?